SU-SWS/saa_victoria_subtheme
# Adapt 381  faux cards cleanup

# READY FOR REVIEW

# Summary
- Add font styles for sub title to go above an h2-h4 in cards
- Added styles for circle headshot image style and square (220px cropped) to be horizontal minimalist faux-cards

# Review By (Date)
- 7/14/20

# Urgency
- Good to move forward with the ADAPT-453 button issue task

# Steps to Test

1. Create page with cards outlined by my steps here: https://stanfordits.atlassian.net/browse/ADAPT-381
1. Create cards for all the first and second card styles
2. Check spacing and font sizes, 
3. check button styles
4. Check responsive
5. Make sure circle and square horizontal cards don't have text wrapping. make sure to follow image style and left-aligned steps to set those up (see jira ticket)
6. Check media and text area large horizontal card and make sure that when it has a subtitle small text, that lines up with the top of the image.
7. Check that space is correct between image and subtitle /preheader for vertical cards.
8. Check that small horizontal cards (square and circle, put in with one text area paragraph) have the pre-heading subtitle aligned with top of image, and the h2 or h3 aligned with the top of the image if there isn't a pre-heading subtitle.

# Affected Projects or Products
- Just SAA subtheme

# Associated Issues and/or People
- JIRA ticket ADAPT-381 ADAPT-453 button issues (started fixing)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
